press/press/doctype/team/suspend_sites.py

```diff
@@ -8,83 +8,62 @@
 
 Defaulters are identified based on the following conditions:
 - Is not a free account
-- Is not an ERPNext Partner
-- Card not added
-- Does not have enough credit balance
+- Is not a Legacy Partner account with payment mode as Partner Credits
+- Has at least one unpaid invoice
+- Has an active site
 
-The `execute` method is the main method which is run by the scheduler daily.
+The `execute` method is the main method which is run by the scheduler on every 10th day of the month.
 """
 
 
 import frappe
-from frappe.utils.data import flt
 
 
 def execute():
-	teams_with_total_usage = get_teams_with_total_usage()
+	teams_with_unpaid_invoices = get_teams_with_unpaid_invoices()
 
-	for d in teams_with_total_usage:
-		total_usage = d.total_usage
+	for d in teams_with_unpaid_invoices:
 		team = frappe.get_doc("Team", d.team)
 
-		if team.free_account or not total_usage or team.get_balance() > 0:
+		if team.free_account or team.payment_mode == "Partner Credits":
 			continue
 
-		total_usage_limit = get_total_free_usage_limit(team)
-
-		# if total usage has crossed the allotted free credits, suspend their sites
-		if total_usage > total_usage_limit:
-			suspend_sites_and_send_email(team)
+		# suspend sites
+		suspend_sites_and_send_email(team)
 
 
 def suspend_sites_and_send_email(team):
-	sites = team.suspend_sites(reason="Card not added and free credits exhausted")
+	sites = team.suspend_sites(reason="Unpaid Invoices")
 	# send email
 	if sites:
 		email = team.user
-		account_update_link = frappe.utils.get_url("/dashboard")
 		frappe.sendmail(
 			recipients=email,
 			subject="Your sites have been suspended on Frappe Cloud",
-			template="payment_failed",
+			template="unpaid_invoices",
 			args={
 				"subject": "Your sites have been suspended on Frappe Cloud",
-				"account_update_link": account_update_link,
-				"card_not_added": True,
 				"sites": sites,
-				"team": team,
 			},
 		)
 
 
-def get_teams_with_total_usage():
-	"""Find out teams which don't have a card, not a free account, not an erpnext partner with their total usage"""
+def get_teams_with_unpaid_invoices():
+	"""Find out teams which has active sites and unpaid invoices and not a free account"""
 	return frappe.db.sql(
 		"""
 		SELECT
-			SUM(i.total) as total_usage,
+			COUNT(i.name) as invoices,
```

Comment:
Is Count needed here? You only need DISTINCT(i.team) in my opinion

cross verify [this](https://frappecloud.com/app/console-log/Log%20on%202023-11-09%2013%3A48%3A26.042993)

Reply:
Yeah I will remove this.